### PR TITLE
Use org.apache.commons.lang3.tuple.Pair instead of javafx.util.Pair

### PR DESCRIPTION
**Pull request by ClarenceThreepwood**
Follow up to #14543

Fixes #14543 

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by bhhari — APPROVED** [review]
thanks for fixing this

File: presto-orc/pom.xml [hunk 1]
```diff
@@ -16,6 +16,12 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.4</version>
```
Comment (bhhari) [inline presto-orc/pom.xml:22]: put the dependency in the parent pom and not have the version here, similar to other dependencies

File: presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java [hunk 2]
```diff
@@ -442,11 +443,11 @@ private final Object createValue(Type type, float filterRate)
         {
             switch (withNulls) {
                 case NONE:
-                    return new Pair<>(false, 1 - filterRate);
+                    return new ImmutablePair<>(false, 1 - filterRate);
```
Comment (mbasmanova) [inline presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java:446]: ImmutablePair.of(x, y) - https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/ImmutablePair.html#of-L-R-

File: presto-orc/pom.xml (hunk 1 above)
Comment (mbasmanova) [reply presto-orc/pom.xml:22]: +1

**Review by mbasmanova — CHANGES_REQUESTED** [review]
@ClarenceThreepwood Thanks for fixing this. Some comments below.

File: presto-orc/pom.xml (hunk 1 above)
Comment (ClarenceThreepwood) [reply presto-orc/pom.xml]: Fixed this in a couple other places too

File: presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java [hunk 3]
```diff
@@ -36,7 +36,8 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import io.airlift.units.DataSize;
-import javafx.util.Pair;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
```
Comment (mbasmanova) [inline presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java:40]: unnecessary import?

**Review by mbasmanova — APPROVED** [review]

**Review by highker — CHANGES_REQUESTED** [review]
We don't actually use `org.apache` in our core codebase. Can we just create a local class for the benchmark rather than introducing a new dep?

File: presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java (hunk 3 above)
Comment (ClarenceThreepwood) [reply presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java:40]: The function returns a Pair - I will change it to return an ImmutablePair

**Comment by ClarenceThreepwood** [comment]
> We don't actually use `org.apache` in our core codebase. Can we just create a local class for the benchmark rather than introducing a new dep?

This package is actually used in a couple of modules - presto-accumulo and presto-parquet. Are you concerned about introducing this as a top-level dependency?

**Comment by highker** [comment]
> Are you concerned about introducing this as a top-level dependency?

Correct. Connectors are classloader safe. We don't really want `org.apache` to appear in main engines. For this commit specifically, let's just create a new private class so that we can avoid the dependency as much as possible.

**Pull request closed without merging**